cli/_cli.php

```diff
@@ -64,3 +67,30 @@ function performRequirementCheck($databaseType) {
 		fail($message);
 	}
 }
+
+function getLongOptions($options, $regex) {
+	$longOptions = array_filter($options, function($a) use ($regex) {
+		return preg_match($regex, $a);
+	});
+	return array_map(function($a) use ($regex) {
+		return preg_replace($regex, '', $a);
+	}, $longOptions);
+}
+
+function validateOptions($input, $params) {
+	$sanitizeInput = getLongOptions($input, REGEX_INPUT_OPTIONS);
+	$sanitizeParams = getLongOptions($params, REGEX_PARAM_OPTIONS);
+	$unknownOptions = array_diff($sanitizeInput, $sanitizeParams);
+
+	if (0 !== count($unknownOptions)) {
+	    $unknownOption = array_shift($unknownOptions);
+	    $possibleOption = array_filter($sanitizeParams, function($a) use ($unknownOption) {
+	        return levenshtein($a, $unknownOption) < 2;
+	    });
+	    if (0 === count($possibleOption)) {
+            fail(sprintf('FreshRSS error: unknown option: %s'), $unknownOption);
+        } else {
+            fail(sprintf('FreshRSS error: unknown option: %s. Do you mean %s?', $unknownOption, array_shift($possibleOption)));
```

Comment:
I'd say "Did you mean" or perhaps "The most similar command is:".

Side note, what if there are multiple results with a low Levenshtein distance?

Reply:
Noted.
In the case of multiple results, the first one is displayed